### DISCUSSION TITLE
Whisper inconsistent with Nova for blank transcripts (Empty alternatives vs Empty words arrays)

### DISCUSSION THREAD

**Discussion by ShantanuNair** [deepgram/community#465] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

I noticed this issue in production, when running on files with audio streams, but no speech detected.
Nova output: 
```
"results": {
        "channels": [
            {
                "alternatives": [
                    {
                        "transcript": "",
                        "confidence": 0,
                        "words": []
                    }
                ]
            }
        ]
    }
```

Whisper output:
```
"results": {
        "channels": [
            {
                "alternatives": []
            }
        ]
    }
```

Expected output would be that it conforms to the same output as Nova when blank, or the other way around. There should be a standardized response for blank transcripts, and IMO a transcript with blank words (as in Nova, above), would be best.

Screenshot demonstrating issue Nova (top) vs Whisper-medium (bottom):
![image](https://github.com/deepgram/community/assets/8559347/a6c78c26-f83f-49e9-9b8d-276b865b027e)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
